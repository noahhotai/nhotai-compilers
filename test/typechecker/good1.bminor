main: function integer ()  = {
    x:integer = 8;
    y: integer = 9;
    z: integer = x+y + 1000 - 50;
}