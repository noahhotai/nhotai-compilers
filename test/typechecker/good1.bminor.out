Program parsed successfully.
x resolves to local 0
y resolves to local 1
Program resolved successfully.
Program type checked successfully.
